engine/src/test/java/io/zeebe/engine/processing/streamprocessor/SkipFailingEventsTest.java

```diff
@@ -384,23 +388,41 @@ public void shouldNotBlacklistInstanceAndIgnoreTimerStartEvents() {
     // given
     when(commandResponseWriter.tryWriteResponse(anyInt(), anyLong())).thenReturn(true);
     final List<Long> processedInstances = new ArrayList<>();
-    final TypedRecordProcessor<TimerRecord> errorProneProcessor =
+    final TypedRecordProcessor<DeploymentRecord> errorProneProcessor =
         new TypedRecordProcessor<>() {
           @Override
           public void processRecord(
-              final TypedRecord<TimerRecord> record,
+              final TypedRecord<DeploymentRecord> record,
               final TypedResponseWriter responseWriter,
               final TypedStreamWriter streamWriter) {
             if (record.getKey() == 0) {
               throw new RuntimeException("expected");
             }
-            processedInstances.add(record.getValue().getProcessInstanceKey());
+            processedInstances.add(TimerInstance.NO_ELEMENT_INSTANCE);
             streamWriter.appendFollowUpEvent(
                 record.getKey(),
                 TimerIntent.CREATED,
                 Records.timer(TimerInstance.NO_ELEMENT_INSTANCE));
           }
         };
+    final BpmnModelInstance process =
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .timerWithDuration("PT1S")
+            .endEvent()
+            .done();
+    final DeploymentRecord firstDeploymentRecord = new DeploymentRecord();
+    firstDeploymentRecord
+        .resources()
+        .add()
+        .setResourceName("process.bpmn")
+        .setResource(Bpmn.convertToString(process).getBytes());
+    final DeploymentRecord secondDeploymentRecord = new DeploymentRecord();
+    secondDeploymentRecord
+        .resources()
+        .add()
+        .setResourceName("process2.bpmn")
+        .setResource(Bpmn.convertToString(process).getBytes());
```

Comment:
We don't need a second deployment record. It is identical to the first one. The resource name doesn't matter for the test. 
We can use the deployment record twice.

Reply:
~The idea here was that we currently rely on broken behavior - when redeploying an existing process, we happen to create the timer. But I believe we want to change this in the future, no? Wouldn't we need a second record at this point to trigger recreating the timer?~

Hm, on second thought, it doesn't matter because we implement the processor ourselves haha :see_no_evil: